src/OSPSuite.R/Services/DataRepositoryTask.cs

```diff
@@ -28,5 +33,28 @@ public void SaveDataRepository(DataRepository dataRepository, string fileName)
       {
          _pkmlPersistor.SaveToPKML(dataRepository, fileName);
       }
+
+      public DataColumn GetErrorColumn(DataColumn column)
+      {
+         if (column.ContainsRelatedColumn(AuxiliaryType.ArithmeticStdDev))
+            return column.GetRelatedColumn(AuxiliaryType.ArithmeticStdDev);
+
+         if (column.ContainsRelatedColumn(AuxiliaryType.GeometricStdDev))
+            return column.GetRelatedColumn(AuxiliaryType.GeometricStdDev);
+
+         return null;
+      }
+
+      public DataColumn AddErrorColumn(DataColumn column, string name = "yError", string errorType = null)
+      {
+         var errorColumn = new DataColumn(name, column.Dimension, column.BaseGrid);
+
+         errorColumn.DisplayUnit = column.DisplayUnit;
+         AuxiliaryType auxiliaryType;
+         errorColumn.DataInfo.AuxiliaryType = Enum.TryParse(errorType, out auxiliaryType) ? auxiliaryType : AuxiliaryType.ArithmeticStdDev;
```

Comment:
Not possible to pass enum from R?
If so: if `Enum.TryParse(errorType, out auxiliaryType)` returns false - I would throw an exception and not (silently) set error to Arithmetic

Reply:
No rClr does not like enums.

The idea was that the erro is arithmetic by default - but now I think it is OK to always require error type and throw an error if an unsupported type is set.

Reply:
TryParse will not throw if unsupported value. It will return false so we can set to the default value. If we rather like the function to throw on unsupported mandatory error type, we need to change the code.

Reply:
I don't have an opinion if it should be mandatory or not. But if it's optional, then these are 2 different use cases for me:
* parameter not passed (null) - use default
* parameter passed - it must be a valid value, otherwise throw exception (at the moment something like `AddErrorColumn(column, "error", "gmetric") `would create an arithmetic error, which is wrong for sure)

Reply:
Agree with Yuri.

Reply:
I also see what you say @Yuri05 and I agree `AddErrorColumn(column, "error", "gmetric")` should not be valid. It is a bit strange for me though to have it optional since it is a string. So, passing null or "" will also be interpreted as default? This is why I proposed not to have it optional anymore but I can live with it optional as long as we agree on what means default behavior. So, optional? if so, should null and/or "" be interpreted as default value?

Reply:
I'm fine to make it mandatory. But as I don't know the requirement for this functionality, I'm fine with optional too.
And if optional: exactly as you say, _should null and/or "" be interpreted as default value_

Reply:
Done, now it is optional and if null or empty will take the default value ;)